monai/transforms/clinical_preprocessing.py

```
@@ -0,0 +1,70 @@
+    """
+    if not isinstance(modality, str):
+        raise TypeError("modality must be a string")
+
+    modality = modality.strip().upper()
+
+    if modality == "CT":
+        transform = get_ct_preprocessing_pipeline()
+    elif modality in ("MR", "MRI"):
+        transform = get_mri_preprocessing_pipeline()
+    else:
+        raise ValueError("Unsupported modality")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)